package exception;

public class Exception1 {
	public static void main(String[] args) {
		int n1, n2, result;
		
		n1 = Integer.parseInt(args[0]);
		n2 = Integer.parseInt(args[1]);
		result = n1/n2;
		 System.out.println("Result "+result);
	}
	

}
